python-cffi/netplan/parser.py

```diff
@@ -46,3 +51,6 @@ def load_nullable_fields(self, input_file: IO):
     def _load_nullable_overrides(self, input_file: IO, constraint: str):
         return _checked_lib_call(lib.netplan_parser_load_nullable_overrides,
                                  self._ptr, input_file.fileno(), constraint.encode('utf-8'))
+
+    def set_flags(self, flags: int):
+        lib.netplan_parser_set_flags(self._ptr, flags)
```

Comment:
suggestion: use a property instead of an actual call

Assuming previous suggestions are implemented, that'd make the following possible:

```py
npp.flags |= Flags.IGNORE_ERRORS
```
rather than
```py
npp.set_flags(saved_flags | NETPLAN_PARSER_FLAGS.IGNORE_ERRORS)
```